constants/constants.ts

```diff
@@ -9,7 +9,14 @@ const daysOfWeek = {
   sat: 6,
 };
 
+const HEADERS_FOR_SSE = {
```

Comment:
SSE abbreviation doesn't give idea on what this constant is about.

Reply:
Probably can add a comment here too like done another place.

Reply:
done✅